packages/components/src/components/button/_mixins.scss

```diff
@@ -26,6 +26,7 @@
   text-decoration: none;
   transition: all $duration--fast-01 motion(entrance, productive);
   outline: $button-outline-width solid transparent;
+  outline-offset: -4px;
```

Comment:
outline-offset is not supported in IE11. Is this an acceptable degradation?

Reply:
I believe so but would like to hear other options. right now the trade off is a workaround for button focus styles in favor of icon only buttons with tooltips

Reply:
I think IE11 is not completely broken it'd be fine, and I guess that's what @elizabethsjudd meant.